app/build.gradle

```diff
@@ -76,11 +76,17 @@ dependencies {
     implementation platform('com.google.firebase:firebase-bom:26.6.0')
     implementation 'com.google.firebase:firebase-firestore-ktx'
 
+    // authentication
+    implementation 'com.google.android.gms:play-services-auth:19.0.0'
+
     implementation "com.google.dagger:hilt-android:$hilt_version"
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation 'com.google.android.gms:play-services-auth:19.0.0'
```

Comment:
Looks like you have this twice in the file

Reply:
Thanks for pointing it out, I will fix it!